cli/src/main/kotlin/commands/DownloaderCommand.kt

```diff
@@ -294,6 +319,60 @@ class DownloaderCommand : CliktCommand(name = "download", help = "Fetch source c
         }
     }
 
+    /**
+     * Determine if a package should be downloaded. A package may not be downloaded if [archiveMode] is used with a
+     * [DownloaderConfiguration] that defines `archivedLicenseCategories`.
+     */
+    private fun shouldBeDownloaded(
+        pkg: Package,
+        ortResult: OrtResult
+    ): Boolean {
+        val archivedLicenseCategories = globalOptionsForSubcommands.config.downloader.archivedLicenseCategories
+
+        if (archiveMode == ArchiveMode.NONE
```

Comment:
Can be dropped when changed as discussed by voice to `includedLicenseCategories`.